src/main/java/org/aludratest/impl/log4testing/Log4TestingAludraTestFramework.java

```diff
@@ -27,6 +27,10 @@
 
     private boolean hooked;
 
+    public Log4TestingAludraTestFramework() {
+        this.hooked = false;
+    }
+
```

Comment:
come on, please don't say that SonarQube wants this... boolean fields are always initialized with false automatically...